test/extended/cli/admin.go

```diff
@@ -0,0 +1,517 @@
+		o.Expect(oc.Run("get").Args("clusterrole/basic-user2").Execute()).To(o.Succeed())
+		o.Expect(oc.Run("label").Args("clusterrole/basic-user2", "foo=bar").Execute()).To(o.Succeed())
+
+		out, err := oc.Run("get").Args("clusterroles", "--selector=foo=bar").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).NotTo(o.ContainSubstring("No resources found"))
+		out, err = oc.Run("get").Args("clusterroles", "--selector=foo=unknown").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("No resources found"))
+
+		o.Expect(oc.Run("get").Args("clusterrolebinding/basic-users2").Execute()).To(o.Succeed())
+		o.Expect(oc.Run("label").Args("clusterrolebinding/basic-users2", "foo=bar").Execute()).To(o.Succeed())
+
+		out, err = oc.Run("get").Args("clusterrolebindings", "--selector=foo=bar").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).NotTo(o.ContainSubstring("No resources found"))
+		out, err = oc.Run("get").Args("clusterrolebindings", "--selector=foo=unknown").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("No resources found"))
+
+		ocns.Run("delete").Args("clusterrole/basic-user2").Execute()
+
+		oc.Run("delete").Args("-f", policyClusterRolesPath).Execute()
+	})
+
+	g.It("ui-project-commands", func() {
+		// Test the commands the UI projects page tells users to run
+		// These should match what is described in projects.html
+		o.Expect(oc.Run("adm", "new-project").Args("ui-test-project", "--admin=createuser").Execute()).To(o.Succeed())
+		o.Expect(oc.Run("adm", "policy", "add-role-to-user").Args("--rolebinding-name=admin", "admin", "adduser", "-n", "ui-test-project").Execute()).To(o.Succeed())
+
+		// Make sure project can be listed by oc (after auth cache syncs)
+		err := wait.Poll(cliInterval, cliTimeout, func() (bool, error) {
+			err := ocns.Run("get").Args("project/ui-test-project").Execute()
+			return err == nil, nil
+		})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// Make sure users got added
+		var out string
+		out, err = oc.Run("get").Args("rolebinding/admin", "-n", "ui-test-project", "-o", "jsonpath={.subjects[*].name}").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("createuser adduser"))
+
+		ocns.Run("delete").Args("project/ui-test-project").Execute()
+	})
+
+	g.It("new-project", func() {
+		// Test deleting and recreating a project
+		o.Expect(oc.Run("adm", "new-project").Args("recreated-project", "--admin=createuser1").Execute()).To(o.Succeed())
+		o.Expect(oc.Run("delete").Args("project", "recreated-project").Execute()).To(o.Succeed())
+		err := wait.Poll(cliInterval, cliTimeout, func() (bool, error) {
+			err := ocns.Run("get").Args("project/recreated-project").Execute()
+			return err == nil, nil
+		})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		o.Expect(oc.Run("adm", "new-project").Args("recreated-project", "--admin=createuser2").Execute()).To(o.Succeed())
```

Comment:
The recreation fails as the previous project was still present:
https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/25169/pull-ci-openshift-origin-master-e2e-gcp/1275050036434046976

Reply:
I'll have to add the unique prefix, probably for the best.

Reply:
I wonder whether just adding `--wait` would be enough here as in the failing test, the project was already being terminated, but wasn't done yet